src/main/java/org/opensearch/performanceanalyzer/decisionmaker/deciders/configs/searchbackpressure/SearchBackPressurePolicyConfig.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.performanceanalyzer.decisionmaker.deciders.configs.searchbackpressure;
+
+
+import java.util.concurrent.TimeUnit;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.performanceanalyzer.decisionmaker.deciders.searchbackpressure.SearchBackPressurePolicy;
+import org.opensearch.performanceanalyzer.rca.framework.core.Config;
+import org.opensearch.performanceanalyzer.rca.framework.core.NestedConfig;
+
+/**
+ * Configures various thresholds for the {@link SearchBackPressurePolicy}
+ *
+ * <p>The config follows the format below "decider-config-settings": {
+ * "search-back-pressure-policy-config": { "enabled": true, // whether the
+ * serch-back-pressure-policy should be enabled "hour-breach-threshold": 30, // threshold for hourly
+ * received unhealthy cluster level rca flow units, if above, then the below thresholds should be
+ * modified, "threshold_count": 1, // how many thresholds to be changed, in this case
+ * search-heap-threshold, "searchbp-heap-stepsize-in-percentage": 5, } }
+ * "searchbp-heap-stepsize-in-percentage" defines the step size to change heap related threshold (in
+ * percentage).
+ */
+public class SearchBackPressurePolicyConfig {
+    private static final Logger LOG = LogManager.getLogger(SearchBackPressurePolicyConfig.class);
+
+    // Field Names
+    private static final String ENABLED = "enabled";
+    private static final String HOUR_BREACH_THRESHOLD = "hour-breach-threshold";
+    private static final String THRESHOLD_COUNT = "threshold_count";
+    private static final String SEARCHBP_HEAP_STEPSIZE_IN_PERCENTAGE =
+            "searchbp-heap-stepsize-in-percentage";
+
+    // Default values
+    public static final boolean DEFAULT_ENABLED = true;
+    // TO DO Decide the Defauilt Hour breach threshold
+    public static final int DEFAULT_HOUR_BREACH_THRESHOLD = 2;
+    public static final int DEFAULT_HOUR_MONITOR_WINDOW_SIZE_MINUTES =
+            (int) TimeUnit.HOURS.toMinutes(1);
+    public static final int DEFAULT_HOUR_MONITOR_BUCKET_SIZE_MINUTES = 1;
```

Comment:
Do we need `DEFAULT` here if these fields are not configurable?

Reply:
Resolved.
Change the naming to HOUR_MONITOR_WINDOW_SIZE_MINUTES and HOUR_MONITOR_BUCKET_SIZE_MINUTES to avoid the confusion with default/configurable values.